packages/page-accounts/src/Sidebar/AddressSection.tsx

```diff
@@ -46,15 +50,36 @@ function AddressSection ({ accountIndex, defaultValue, editingName, flags, onCha
           value={value}
           withSidebar={false}
         />
+
         <div className='ui--AddressMenu-addr'>
           {value}
         </div>
+
         {accountIndex && (
           <div className='ui--AddressMenu-index'>
             <label>{t<string>('index')}:</label> {accountIndex}
           </div>
         )}
       </div>
+
+      <div className='ui--AddressSection__CopyColumn'>
+        <div className='ui--AddressMenu-copyaddr'>
+          <CopyToClipboard
+            text={value}
+          >
+            <span>
+              <Button.Group>
+                <Button
+                  icon={isCopyShown ? 'check' : 'copy'}
+                  label={wasCopied ? t<string>('Copy') : t<string>('Copy')}
+                  onClick={toggle}
+                  onMouseLeave={isCopyShown ? toggle : donothing }
```

Comment:
What if we write in that way: `onMouseLeave={isCopyShown && toggle}`?
In that case, we won't need to have that hook: `const [wasCopied, donothing] = useToggle(false);`

Reply:
removed the wasCopied variable which was useless